### DISCUSSION TITLE
Control our pc?

### DISCUSSION THREAD

**Discussion by VasigaranAndAngel** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#460] [General]
What if this application can control our pc?
Like shutting down or locking the pc, Opening any application, Media player controls, etc...

**Comment by sumeshir26** [comment]
@VasigaranAndAngel This is on the roadmap and will be added soon.

**Reply by VasigaranAndAngel** [reply]
Wow, that's cool. I'm waiting...
